src/pages/questions/ProductInformation.jsx

```diff
@@ -120,18 +170,41 @@ const ProductInformation = ({ question }) => {
       />
       {!hideImages && productData?.images && (
         <Grid container rowSpacing={1.5} spacing={1}>
-          {getImagesUrls(productData.images, question.barcode).map((src) => (
-            <Grid item key={src}>
-              <Zoom>
-                <img
-                  src={src}
-                  alt=""
-                  loading="lazy"
-                  style={{ maxWidth: 300, maxHeight: 300 }}
-                />
-              </Zoom>
-            </Grid>
-          ))}
+          {getImagesUrls(productData.images, question.barcode).map(
+            (src, imgid) => (
+              <Grid
+                item
+                key={src}
+                style={{ display: "inline-flex", alignItems: "flex-start" }}
+              >
+                <Zoom>
+                  <img
+                    src={src}
+                    alt=""
+                    loading="lazy"
+                    style={{ maxWidth: 300, maxHeight: 300 }}
+                  />
+                </Zoom>
+                {flagged[src[src.length - 5]] ? (
```

Comment:
If `imgid` becomes reliable
```suggestion
                {flagged[imgid] ? (
```

Reply:
this could be `flagged.includes(getImageId(src))`, please check the comment above